src/docs/get-started/install/_get-sdk.md

```diff
@@ -111,6 +81,30 @@ process.
 Once you have installed any missing dependencies, run the `flutter doctor`
 command again to verify that you’ve set everything up correctly.
 
+### Downloading straight from GitHub instead of using a tarball
+
+_This is only suggested for advanced use cases._
+
+You can also use git directly instead of downloading the prepared tarball. For example,
```

Comment:
Should we use something more generic like "archive" over "tarball"? I believe we use zip on mac and windows.

Reply:
done